src/components/description-list/_macro-options.md

```diff
@@ -1,11 +1,12 @@
-| Name                 | Type          | Required | Description                                                                                                    |
-| -------------------- | ------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
-| id                   | string        | false    | The HTML `id` for the description list (`<dl>`)                                                                |
-| classes              | string        | false    | Classes to apply to description list (`<dl>`)                                                                  |
-| descriptionListLabel | string        | false    | Sets the HTML `title` and `aria-label` attributes to provide additional information about the description list |
-| termCol              | number        | true     | The number of grid columns used for the `<dt>` elements above medium breakpoint                                |
-| descriptionCol       | number        | true     | The number of grid columns used for the `<dd>` elements above medium breakpoint                                |
-| itemsList            | array`<Item>` | true     | Settings for the terms and descriptions of the [description list items](#item)                                 |
+| Name                 | Type            | Required | Description                                                                                                    |
+| -------------------- | --------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
+| id                   | string          | false    | The HTML `id` for the description list (`<dl>`)                                                                |
+| classes              | string          | false    | Classes to apply to description list (`<dl>`)                                                                  |
+| descriptionListLabel | string          | false    | Sets the HTML `title` and `aria-label` attributes to provide additional information about the description list |
+| termCol              | number          | true     | The number of grid columns used for the `<dt>` elements above medium breakpoint                                |
+| descriptionCol       | number          | true     | The number of grid columns used for the `<dd>` elements above medium breakpoint                                |
+| itemsList            | array`<Item>`   | true     | Settings for the terms and descriptions of the [description list items](#item)                                 |
+| variants             | array or string | false    | An array of values or single value (string) to adjust the component using available variants:“inline”          |
```

Comment:
We only have one variant at the moment so this should just be called "variant" and should only accept a string